src/routes/manage_job/get.rs

```diff
@@ -1,11 +1,37 @@
+use std::convert::{TryFrom, TryInto};
+
 use crate::errors::ReacherError;
-use sqlx::{Pool, Postgres};
+
+use csv::WriterBuilder;
+use sqlx::{Executor, Pool, Postgres, Row};
 use warp::Filter;
 
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 
 use sqlx::types::chrono::{DateTime, Utc};
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum JobResultResponseFormat {
+	Json,
+	Csv,
+}
+
+// limit and offset are optional in the request
+// if they are unspecified their default values
+// are 50 and 0 respectively
+#[derive(Serialize, Deserialize)]
+struct JobResultRequest {
+	format: JobResultResponseFormat,
```

Comment:
I think we should make this query param optional too, and default to JSON

Reply:
Done